### ISSUE TITLE
Invalid exception code 

### ISSUE THREAD

**Issue by AdrienHerubel** [skelsec/minidump#16]
I hit the following error when parsing a minidump file  : 

`ValueError: 3762504530 is not a valid ExceptionCode`

I tried adding this exception code to the enum (0xE0434352) but then I hit another. I had to add the following three exceptions :
```
EXCEPTION_000 = 0xE0434352
EXCEPTION_001 = 0x6AA71151
EXCEPTION_002 = 0x716BF6D8
````

Alternatively the following changes ignore unknown exception codes : 
```
		try :
			me.ExceptionCode = ExceptionCode(int.from_bytes(buff.read(4), byteorder = 'little', signed = False))
		except :
			me.ExceptionCode = ExceptionCode.EXCEPTION_NONE
```

**Comment by skelsec** [comment]
I have a fix in the `speedup` branch, but it needs more testing. Will merge it this week (hopefully).
Thank you for the report!

**Comment by skelsec** [comment]
This issue is considered fixed in the latest release `0.0.17`.
